pints/_abc/_abc_rejection.py

```diff
@@ -0,0 +1,83 @@
+#
+# ABC Rejection method
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+import pints
+
+
+class RejectionABC(pints.ABCSampler):
+    r"""
+    Implements the rejection ABC algorithm as described in [1].
+
+    Here is a high-level description of the algorithm:
+
+    .. math::
+        \begin{align}
+        \theta^* &\sim p(\theta) \\
+        x &\sim p(x|\theta^*) \\
+        \textrm{if } s(x) < \textrm{threshold}, \textrm{then} \\
+        \theta^* \textrm{ is added to list of samples} \\
+        \end{align}
+
+    In other words, the first two steps sample parameters
+    from the prior distribution and then sample data from the
+    sampling distribution (assuming the sampled parameters).
+    In the end, if the summary statistics are within the threshold,
```

Comment:
Rather than use the term summary statistic, which is nowhere else used in the code or docs, I would explain this in terms of the `error_measure`